code/game/objects/items/weaponry.dm

```diff
@@ -586,6 +586,34 @@
 	flags_1 = DROPDEL_1 | ABSTRACT_1
 	attack_verb = list("bopped")
 
+/obj/item/slapper
+	name = "slapper"
+	desc = "This is how real men fight."
+	icon_state = "latexballon"
+	force = 1
+	throwforce = 0
+	flags_1 = DROPDEL_1 | ABSTRACT_1
+	attack_verb = list("slapped")
+	hitsound = 'sound/effects/snap.ogg'

```

Comment:
we used to have slapping is this the right sound file?

Reply:
yes